ui/server/graphql/resolvers/helpers.ts

```diff
@@ -324,3 +355,14 @@ export async function roundMemberBalance(member) {
 
   return totalAllocations - totalContributions;
 }
+
+/** only call this if you've verified the user is at least a round admin */
+export async function stripeIsConnected({ round }) {
+  if (!round.stripeAccountId) {
+    return false;
+  }
+
+  const account = await stripe.accounts.retrieve(round.stripeAccountId);
```

Comment:
Note that this will slightly slow down page loads for admins since this is called in `TOP_LEVEL_QUERY` and is an api call to stripe's servers. The slowdown might not be noticeable though. In the future if we want to speed it up we can move the parts of the graphql query that's relevant to the settings into the settings page.

Reply:
i ended up doing this anyway because of a permissions bug